lib/core/network/repository/profile_repository.dart

```diff
@@ -1,15 +1,17 @@
+import 'package:get_it/get_it.dart';
 import 'package:hypha_wallet/core/error_handler/model/hypha_error.dart';
 import 'package:hypha_wallet/core/error_handler/model/hypha_error_type.dart';
 import 'package:hypha_wallet/core/network/api/endpoints.dart';
+import 'package:hypha_wallet/core/network/api/remote_config_serivice.dart';
 import 'package:hypha_wallet/core/network/networking_manager.dart';
 import 'package:hypha_wallet/ui/architecture/result/result.dart';
 import 'package:hypha_wallet/ui/profile/interactor/profile_data.dart';
 
 class ProfileService extends NetworkingManager {
-  ProfileService() : super(Endpoints.profileServiceEndpoint);
+  ProfileService() : super(GetIt.I<RemoteConfigService>().profileServiceEndpoint);
```

Comment:
You need to inject this RemoteConfigService here. 
in the `_registerServicesModule`  replace _registerLazySingleton<ProfileService>(() => ProfileService()); with 
 _registerLazySingleton<ProfileService>(() => ProfileService(_getIt<RemoteConfigService>()));

Reply:
Add a RemoteConfigService  class instance field here 

final RemoteConfigService remoteConfig;

inside the constructor add required this.remoteConfig

Reply:
then you can use the remoteConfig instead of doing GetIt.I<RemoteConfigService>() each time